13.yaml

```
@@ -0,0 +1,59 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: q13
+  labels:
+    question.kontest.amsy.dev: "13"
+data:
+  question: |-
+    Kubernetesでは、Podの一時ディスク領域として利用できる機能があります。Podのdeleteと同時にその領域も削除されます。
```

```suggestion
    emptyDirを使うと、Podに対して一時的なディスク領域をマウントすることができます。この領域は、Podのdeleteと同時に削除されます。
```

一時ディスクってAzureくらいでしか使われないマイナーな表現かなと思います

---

一応問題ということで、emptyDirとあからさまに書いてしまうとと思ったので、遠回しな表現にしていました。
ヒント見ればというところもありますが。もしなんかうまい表現が無ければ、そのままemptyDirにしましょうか。

---

一時ディスク領域（一時的なディスク領域）の表現自体はいいと思います。一般名詞的な使い方なので。
なので遠回しな表現自体には酸性です。

ただ、CSIのInline PVや通常のPod領域も同じように使えそうなので、答えの一意性という面で悩ましいですね…。
Empty Dirとかにしておきますかね。。

---

Empty Dirとしましょう。

---

```suggestion
    Empty Dirを使うと、Podに対して一時的なディスク領域をマウントすることができます。この領域は、Podのdeleteと同時に削除されます。
```

こうかな

---

その方向で修正します。